x/ccv/provider/keeper/keeper.go

```diff
@@ -1190,14 +1190,17 @@ func (k Keeper) SetOptedIn(
 	chainID string,
 	providerAddr types.ProviderConsAddress,
 	blockHeight uint64,
+	power uint64,
```

Comment:
Introduce `Power` in the opted-in state so we can see if we should send a `ValidatorUpdate` down to CometBFT based on whether the power changed since last time it was sent. Only makes sense in an epoch-based setting because in a non-epoch-based setting, we could use the validator updates from the `staking` module.